app/models/tag_mapping.rb

```diff
@@ -0,0 +1,6 @@
+class TagMapping < ActiveRecord::Base
```

Comment:
Reworked names a bit.

A TaggingSpreadsheet has many TagMappings. Keeping the latter because its reasonably clear. 

Only stuff in a namespace is the service objects.